### ISSUE TITLE
Maps application where you can set up appointments geographically

### ISSUE THREAD

**Issue by Kreijstal** [open-source-ideas/open-source-ideas#147]
## Project description

[_Describe the project the best you can. Give any background information or link to resources that are necessary to understand the problem it is intended to solve. The more you elaborate on your idea, the easier it is to accomplish._]

It could also be a plugin of some mayor app
## Relevant Technology

[_Write what technology is relevant. What language, what platform, any particular library/framework/existing project it is based on?_]

## Complexity and required time

[_Please only tick off one box in each category by changing `[ ]` to `[x]`. The labels on the project will then be updated by the maintainers as soon as possible._]

### Complexity

- [ ] Beginner - This project requires no or little prior knowledge of the technolog(y|ies) specified to contribute to the project
- [ ] Intermediate - The user should have some prior knowledge of the technolog(y|ies) to the point where they know how to use it, but not necessarily all the nooks and crannies of the technology
- [x] Advanced - The project requires the user to have a good understanding of all components of the project to contribute

### Required time (ETA)

- [ ] Little work - A couple of days
- [ ] Medium work - A week or two
- [x] Much work - The project will take more than a couple of weeks and serious planning is required

**Comment by Thrilleratplay** [comment]
> Maps application where you can set up appointments geographically

Based on the title, I was going to point out the traveling sales man problem.  I would reword the title, it sounds like you are suggesting travel time augmented calendar reminders based on location and mode of transportation.  I like the idea but it would also need to [account for traffic where available](http://opentransportmap.info/) and maybe bus schedules or extreme weather conditions if walking.

**Comment by FiddleINC** [comment]
This is a cool idea. I really like to work on it.
